### ISSUE TITLE
[a11y]: Cannot navigate to content in EmptyState component

### ISSUE THREAD

**Issue by laramlewis** [carbon-design-system/ibm-products#6992]
### Package

@carbon/ibm-products

### Browser

Chrome

### Operating System

MacOS

### Package version

2.59.0

### React version

20.18.3

### Automated testing tool and ruleset

Manual and screen reader testing

### Assistive technology

VoiceOver

### Description

I expected to be able to navigate to the content within the EmptyState components with the keyboard or VO commands so the screen reader could read the content. Without this, visually impaired users will not be able to access this content.

### WCAG 2.1 Violation

2.1.1 Keyboard

### Reproduction/example

https://ibm-products.carbondesignsystem.com/?path=/docs/ibm-products-patterns-empty-state-emptystate--docs&globals=viewport:basic

### Steps to reproduce

Using the storybook pages for the EmptyState components: https://ibm-products.carbondesignsystem.com/?path=/docs/ibm-products-patterns-empty-state-emptystate--docs&globals=viewport:basic turn on VoiceOver and see that there is no way to navigate to the content within the EmptyState component so the screen reader can read the content. This appears to be the case for all of the components. E.g. EmptyState, ErrorEmptyState, NoDataEmptyState, etc.

### Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/ibm-products/blob/main/.github/CODE_OF_CONDUCT.md)
- [x] I checked the [current issues](https://github.com/carbon-design-system/ibm-products/issues) for duplicate problems

**Comment by wkeese** [comment]
It works for me (using ctrl-option-arrow keys).

https://github.com/user-attachments/assets/543e5efd-d928-4060-ba46-5cf4390be515

**Comment by laramlewis** [comment]
Thanks, @wkeese. Upon second look you are correct. I can access the content if I enter Ctrl+Option+Shift+Down Arrow twice after the group is selected so the problem must be within our use of this code. I will close this issue and open one in our tracker
